agentic-ai/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Jakarta Agentic Artificial Intelligence"
+summary: "The Jakarta Agentic AI project provides a set of vendor-neutral APIs that make it easy, consistent, and reliable to build, deploy, and run AI agents on Jakarta EE runtimes."
+summary_sixty_char: "Jakarta API for Agentic AI"
+project_id: "ee4j.agentic-ai"
```

Comment:
Is this project expected to be at https://projects.eclipse.org/ee4j.agentic-ai? It's 404 now.

Reply:
This results in _Could not load project_:
<img width="800" height="73" alt="image" src="https://github.com/user-attachments/assets/ac2a822c-aba7-4b65-9a9e-782b82df2733" />